src/utils/transformCommonComponents.ts

```diff
@@ -0,0 +1,84 @@
+import { CommonComponents } from '@prisma/client';
+
+function tranformCommonComponentOption(original) {
```

Comment:
I see the same helper function `tranformCommonComponentOption` in one of the migration script.
Is there any chance that we can use this in the migration script to reduce duplication?